ush/gldas_get_data.sh

```diff
@@ -0,0 +1,76 @@
+#! /usr/bin/env bash
+#########################################################
+# This script generate gldas forcing from gdas prod sflux
+#########################################################
+
+source "${HOMEgfs:?}/ush/preamble.sh"
+
+bdate=$1
+edate=$2
+
+if [[ "${USE_CFP:-"NO"}" = "YES" ]] ; then
+  touch ./cfile
+fi
+
+### COMINgdas = prod gdas sflux grib2
+### RUNDIR = gldas forcing in grib2 format
+### RUNDIR/force = gldas forcing in grib1 format
+export COMPONENT=${COMPONENT:-atmos}
+fpath=${RUNDIR:?}
+gpath=${RUNDIR}/force
+cycint=${assim_freq:-6}
+
+# get gdas flux files to force gldas.
+# CPC precipitation is from 12z to 12z. One more day of gdas data is
+# needed to disaggregate daily CPC precipitation values to hourly values
+cdate=$(${NDATE:?} -12 "${bdate}")
+
+iter=0
+
+#-------------------------------
+while [[ "${cdate}" -lt "${edate}" ]]; do
+#-------------------------------
+  ymd=$(echo "${cdate}" |cut -c 1-8)
+  cyc=$(echo "${cdate}" |cut -c 9-10)
+  [[ ! -d ${fpath}/gdas.${ymd} ]] && mkdir -p "${fpath}/gdas.${ymd}"
+  [[ ! -d ${gpath}/gdas.${ymd} ]] && mkdir -p "${gpath}/gdas.${ymd}"
+
+f=1
```

Comment:
This is still part of the while block but lost its indentation.

Reply:
Indented.